extensions/standard-processors/processors/TailFile.h

```diff
@@ -184,11 +199,17 @@ class TailFile : public core::ProcessorImpl {
       .withAllowedTypes<minifi::controllers::AttributeProviderService>()
       .build();
   EXTENSIONAPI static constexpr auto BatchSize = core::PropertyDefinitionBuilder<>::createProperty("Batch Size")
-      .withDescription("Maximum number of flowfiles emitted in a single trigger. If set to 0 all new content will be processed.")
+      .withDescription("Maximum number of lines emitted in a single trigger. If set to 0 all new content will be processed.")
       .isRequired(true)
       .withValidator(core::StandardPropertyValidators::UNSIGNED_INTEGER_VALIDATOR)
       .withDefaultValue("0")
       .build();
+  EXTENSIONAPI static constexpr auto ResultFormat = core::PropertyDefinitionBuilder<magic_enum::enum_count<TailResultFormat>()>::createProperty("Result Mode")
+      .withDescription("Specifies how the result lines are arranged into output flow files")
+      .isRequired(true)
+      .withDefaultValue(magic_enum::enum_name(TailResultFormat::FlowFilePerDelimiter))
+      .withAllowedValues(magic_enum::enum_names<TailResultFormat>())
+      .build();
```

Comment:
PROCESSORS.md should be updated with these changes

Reply:
nice catch, updated it in https://github.com/apache/nifi-minifi-cpp/pull/1972/commits/20629773e81a1f89649cd68274bb44ff9568d0b9